ably-ios/ARTTypes.h

```diff
@@ -47,9 +47,7 @@ typedef NS_ENUM(NSUInteger, ARTRealtimeChannelState) {
     ARTRealtimeChannelInitialised,
     ARTRealtimeChannelAttaching,
     ARTRealtimeChannelAttached,
-    ARTRealtimeChannelDetaching,
```

Comment:
State `Detaching` exists, see http://docs.ably.io/client-lib-development-guide/features/#RTL2a, why was this removed?